src/libData/AccountData/AccountStore.h

```diff
@@ -68,8 +68,7 @@ class AccountStoreTemp : public AccountStoreSC<std::map<Address, Account>> {
 
 // Singleton class for providing interface related Account System
 class AccountStore
-    : public AccountStoreTrie<std::unordered_map<Address, Account>>,
```

Comment:
I also don't like that Singleton, but is removing it absolutely necessary in this PR?  Would it not cause some unknown problems somewhere where AccountStore singleton is used?

Reply:
As discussed, it is not used and overwritten by the AccountSore::GetInstance